### ISSUE TITLE
[Question] training with higher resolution 

### ISSUE THREAD

**Issue by dill-shower** [IceClear/StableSR#31]
Hello. I own an A100 and am about to train my own stableSR model for non-realistic images. Since the video card has 80gb vram, can I use a resolution other than 512 for training? For example 1024 or higher. It seems that i MUST to use sd2.1-512 as a stable-diffusion model and can't use 768-v, so this question arose. Does it make sense to change the resolution in config to something higher than 512 or will it only worsen the result?

**Comment by IceClear** [comment]
Hi. I tried other models and I think you can try to fine-tune on other SD models such as 768-v. You even do not need to change the loss function. For different resolutions, I am not sure if a higher resolution makes sense. You may have a try.

**Comment by dill-shower** [comment]
Thanks, I will try
